pydp/algorithms/_algorithm.py

```diff
@@ -105,19 +105,7 @@ def reset(self):
         """
         return self.__algorithm.reset()
 
-    def serialize(self):
```

Comment:
Can we comment it out rather than removing it?

Reply:
Done